testsuite/MDAnalysisTests/analysis/test_contacts.py

```diff
@@ -302,6 +303,26 @@ def test_non_callable_method(self, universe):
         with pytest.raises(ValueError):
             self._run_Contacts(universe, method=2, stop=2)
 
+    @pytest.mark.parametrize("pbc", (True, False))
+    def test_distance_box(self, pbc):
+        u = mda.Universe(TPR, XTC)
+        sel_basic = "(resname ARG LYS)"
+        sel_acidic = "(resname ASP GLU)"
+        acidic = u.select_atoms(sel_acidic)
+        basic = u.select_atoms(sel_basic)
+        
+        r = contacts.Contacts(u, select=(sel_acidic, sel_basic),
+                        refgroup=(acidic, basic), radius=6.0, pbc=pbc)
+        r.run()
+        if pbc:
+            expected = [1., 0.43138152, 0.3989021, 0.43824337, 0.41948765,
```

Comment:
Looks good @ss62171 -- personally I would change the test so that the expected values are also part of `parametrize`, e.g.

```python
@pytest.mark.parametrize("pbc,expected", [
    
    (True, [1., 0.43138152, 0.3989021, 0.43824337, 0.41948765,
            0.42223239, 0.41354071, 0.43641354, 0.41216834, 0.38334858]),
    (False, [1., 0.42327791, 0.39192399, 0.40950119, 0.40902613,
             0.42470309, 0.41140143, 0.42897862, 0.41472684, 0.38574822])
    
    ])
```

A new push would also restart the Appveyor tests, which as far as I can tell have failed due to running out of time. I can't seem to restart these jobs individually without a commit -- @MDAnalysis/coredevs have any insight?